components/ui/authbutton.tsx

```diff
@@ -14,25 +15,40 @@ export default async function AuthButton() {
   };
 
   return (
-    <div className="hidden md:flex">
+    <div className="hidden flex-row items-center space-x-1 md:flex">
```

Comment:
- fix spacing `space-x-4` instead of 1

Reply:
already done as i see it i think u did it someone